src/app/Components/Label/LabelDemo.tsx

```diff
@@ -1,117 +1,33 @@
 import React from 'react';
 import {
-  Dropdown,
-  DropdownItem,
-  DropdownList,
-  MenuToggle,
-  MenuToggleElement,
+  Label,
+
   PageSection,
-  Title,
   Text,
-  Alert,
   TextContent,
-  Divider,
 } from '@patternfly/react-core';
-import Button from '@mui/material/Button';
-import Menu from '@mui/material/Menu';
-import MenuItem from '@mui/material/MenuItem';
-import KeyboardArrowDownIcon from '@mui/icons-material/KeyboardArrowDown';
+import Chip from '@mui/material/Chip';
 
 export const LabelDemo: React.FunctionComponent = () => {
 
-  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
-  const open = Boolean(anchorEl);
-  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
-    setAnchorEl(event.currentTarget);
-  };
-  const handleClose = () => {
-    setAnchorEl(null);
-  };
-
-  const [isOpen, setIsOpen] = React.useState(false);
-
-
-  const onToggleClick = () => {
-    setIsOpen(!isOpen);
-  };
-
-  const onSelect = (_event: React.MouseEvent<Element, MouseEvent> | undefined, value: string | number | undefined) => {
-    // eslint-disable-next-line no-console
-    console.log('selected', value);
-    setIsOpen(false);
-  };
-
   return (
     <PageSection>
       <TextContent>
-        <Text component="h1">Dropdown demo </Text>
+        <Text component="h1">Label demo </Text>
         <Text component="p">
-          Dropdown theming demo. This is a small demo to test theming the PF6 Alert component. Stylesheet contains
-          themed tokens for PF6 component and has been loaded into this demo.
+          Label theming demo. This is a small demo to test theming the PF6 Label component. Stylesheet contains themed
+          tokens for PF6 component and has been loaded into this demo.
         </Text>
       </TextContent>
       <br />
-      MUI Dropdown:
-
+      MUI Label:
       <div>
-      <Button
-        id="basic-button"
-        aria-controls={open ? 'basic-menu' : undefined}
-        aria-haspopup="true"
-        aria-expanded={open ? 'true' : undefined}
-        onClick={handleClick}
-        endIcon={<KeyboardArrowDownIcon />}
-        disableRipple
-      >
-        Dashboard
-      </Button>
-      <Menu
-        id="basic-menu"
-        anchorEl={anchorEl}
-        open={open}
-        onClose={handleClose}
-        MenuListProps={{
-          'aria-labelledby': 'basic-button',
-        }}
-      >
-        <MenuItem onClick={handleClose}>Profile</MenuItem>
-        <MenuItem onClick={handleClose}>My account</MenuItem>
-        <MenuItem onClick={handleClose}>Logout</MenuItem>
-      </Menu>
-    </div>
+        <Chip label="abc" color="primary" size="small" />
```

Comment:
Thank you @srambach , I pulled in / am referencing the PF `Label` component below. MUI uses `Chip` instead of "Label", whereas PF now uses `Label` instead of `Chip`.